### PR TITLE
New hairstyle! Short bedhead.

### PR DESCRIPTION
**Pull request by Adrian16199**
A new barber has arrived and made a new hairstyle for people to go for!

## About the PR

Adds new hairstyle into the game: Short bedhead.

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: A new barber has been seen nearby of your station! Seems to have a new hairstyle to show.

### CODE REVIEW

File: Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json
Comment (DEATHB4DEFEAT) [inline Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json:1]: JSON minifying makes conflicts and future reviews much harder, format this readably

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json
Comment (Adrian16199) [reply Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json:1]: Like this?

Comment (DEATHB4DEFEAT) [reply Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json:1]: yeah

File: Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                [
+                    1.0
+                ],
+                [
+                    1.0
+                ]
+            ]
+        }
+    ]
+}
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/DeltaV/Mobs/Customization/new_hair.rsi/meta.json]:
```suggestion
}

```

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Comment by Adrian16199** [comment]
Is there a particular reason why it failed "Build & Test Debug / build (windows-latest) (pull_request)"?

**Comment by DEATHB4DEFEAT** [comment]
> Is there a particular reason why it failed "Build & Test Debug / build (windows-latest) (pull_request)"?

not related

**Comment by DebugOk** [comment]
> * I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Where?

**Comment by Adrian16199** [comment]
> > * I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
> 
> Where?

I did include a png while makin this post but its not appearing and i have no idea why. I will probably just edit it that i didnt if i can.

**Comment by Adrian16199** [comment]
Hopefully it says now that i did not include any screenshots and such so there is no more issues.

**Pull request merged**